cmd/start.go

```diff
@@ -173,3 +181,53 @@ func Start(c *cli.Context) error {
 		return fmt.Errorf("unexpected context cancellation")
 	}
 }
+
+func initializeMetrics(
+	ctx context.Context,
+	config *config.Config,
+	netProvider net.Provider,
+	stakeMonitor chain.StakeMonitor,
+) error {
```

Comment:
Is it possible we'll ever get `error` from here? I think I missed it when reviewing https://github.com/keep-network/keep-core/pull/1850/

Reply:
Yeah, currently not.

Reply:
Can we drop this `error` and do a quick PR for `keep-core` as well?

Reply:
Sure